### DISCUSSION TITLE
Option for sidebar with tree (list) view?

### DISCUSSION THREAD

**Discussion by JayBrown** [mikekazakov/nimble-commander#236] [Ideas]
I've been looking into Rixstep's Xfile software, and a good thing to have would be an (optional!) sidebar with a tree (list) view of the volume, drop down menus for quick navigation, toggle hidden files in sidebar (default on). There could possibly be a better way to access external volumes in the sidebar, instead of going through /Volumes/:

Screenshot of Xfile sidebar:
![RixstepXfileSidebar](http://i.imgur.com/RAB9xvA.png)

Note: migrated here from the magnumbytes forum

**Comment by singalen** [comment]
Might be a panel as well.
